test/compose-pipe-style.js

```diff
@@ -0,0 +1,50 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/compose-pipe-style';
+
+const ruleTester = avaRuleTester(test, {
+    env: {
+        es6: true
+    },
+    parserOptions: {
+        sourceType: 'module'
+    }
+});
+
+const error = {
+    compose: {
+        ruleId: 'compose-pipe-style',
+        message: 'Prefer `pipe` over `compose` for multiline expression'
+    },
+    pipe: {
+        ruleId: 'compose-pipe-style',
+        message: 'Prefer `compose` over `pipe` for single line expression'
+    }
+};
+
+ruleTester.run('compose-pipe-style', rule, {
+    valid: [
+        'compose(a, b, c)',
+        'R.compose(a, b, c)',
+        'pipe(\na,\nb,\nc\n)',
+        'R.pipe(\na,\nb,\nc\n)',
```

Comment:
Actually not every parameter needs to be on its own line. The rule also allows `R.pipe(\na, b, c);` or `R.pipe(a, b, c\n);`

Reply:
Just for the record: something like

```
R.compose(
    a, b, c
);
```

is not allowed. Is this still what you would expect from the rule @jogold?

Reply:
For `pipe`: will update to disallow this and add test cases
For `compose`: yes, will update to add test cases

Reply:
For `pipe`: this can be covered by the default of [`function-paren-newline`](https://eslint.org/docs/rules/function-paren-newline), do you think this rule needs to enforce that all arguments have to bee on separate lines?
For `compose`: yes, this is what was intended (can also covered by `function-paren-newline`), I can add test cases to reflect that this was the intent.

Reply:
> do you think this rule needs to enforce that all arguments have to be on separate lines?

That is what I actually wanted to clarify. For me it would be ok to not do this in this rule and let other rules handle it. But adding tests would be nice to document what the intended behavior is.